### PR TITLE
Exporting and downloading of file bug fix.

### PR DESCRIPTION
**Pull request by jnmrclmbsjse**
This PR includes:
1. Changed exporting from overlay to ajax call.
2. Auto download on file generation.
3. Fallback download button if popup is blocked.
4. Manual export via CLI guide.

### CODE REVIEW

File: src/StoreKeeper/WooCommerce/B2C/Endpoints/FileExport/ExportEndpoint.php [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+            'max_execution_time',
+            60 * 60 * 12, // Time in hours,
+            [$this, 'throwIniError']
+        );
+
+        IniHelper::setIni(
+            'memory_limit',
+            '512M',
+            [$this, 'throwIniError']
+        );
```
Comment (ikulis) [inline src/StoreKeeper/WooCommerce/B2C/Endpoints/FileExport/ExportEndpoint.php:30]: We should allow the export even though the Ini cannot be set to those values

File: src/StoreKeeper/WooCommerce/B2C/Backoffice/Pages/Tabs/ExportTab.php [hunk 2]
```diff
@@ -14,37 +16,50 @@
...
+                'Your download will start in a few seconds. If not, you can download the file manually using the link below' => __('Your download will start in a few seconds. If not, you can download the file manually using the link below', I18N::DOMAIN),
+                'Please wait and keep the page and popup window open while we are preparing your export' => __('Please wait and keep the page and popup window open while we are preparing your export', I18N::DOMAIN),
+                'Preparing export' => __('Preparing export', I18N::DOMAIN),
+                'Stop exporting' => __('Stop exporting', I18N::DOMAIN),
+                'Size' => __('Size', I18N::DOMAIN),
+                'Export failed' => __('Export failed', I18N::DOMAIN),
+                'Something went wrong during export or server timed out. You can try manual export via command line, do you want to read the guide?' => __('Something went wrong during export or server timed out. You can try manual export via command line, do you want to read the guide?', I18N::DOMAIN),
+                'No, thanks' => __('No, thanks', I18N::DOMAIN),
+                'Yes, please' => __('Yes, please', I18N::DOMAIN),
```
Comment (ikulis) [inline src/StoreKeeper/WooCommerce/B2C/Backoffice/Pages/Tabs/ExportTab.php:57]: those translations should be escaped, cos you directly show the in html

File: src/StoreKeeper/WooCommerce/B2C/Backoffice/Pages/Tabs/ExportTab.php [hunk 3]
```diff
@@ -146,74 +160,55 @@ private function renderExport()
...
+                    sprintf(__('Run %s to export %s.', I18N::DOMAIN), '<code>wp sk file-export-tag</code>', __('tags')),
+                    sprintf(__('Run %s to export %s.', I18N::DOMAIN), '<code>wp sk file-export-category</code>', __('categories')),
+                    sprintf(__('Run %s to export %s.', I18N::DOMAIN), '<code>wp sk file-export-attribute</code>', __('attributes')),
+                    sprintf(__('Run %s to export %s.', I18N::DOMAIN), '<code>wp sk file-export-attribute-option</code>', __('attribute options')),
+                    sprintf(__('Run %s to export %s.', I18N::DOMAIN), '<code>wp sk file-export-product-blueprint</code>', __('product blueprints')),
+                    sprintf(__('Run %s to export %s.', I18N::DOMAIN), '<code>wp sk file-export-product</code>', __('products')),
+                ],
+            ],
+            __('Each run will return the path of the exported file.', I18N::DOMAIN),
+        ];
```
Comment (ikulis) [inline src/StoreKeeper/WooCommerce/B2C/Backoffice/Pages/Tabs/ExportTab.php:185]: those translations should be escaped, cos you directly show the in html

File: src/StoreKeeper/WooCommerce/B2C/Endpoints/FileExport/ExportEndpoint.php (hunk 1 above)
Comment (jnmrclmbsjse) [reply src/StoreKeeper/WooCommerce/B2C/Endpoints/FileExport/ExportEndpoint.php]: Used `logIniError` instead.

File: src/StoreKeeper/WooCommerce/B2C/Backoffice/Pages/Tabs/ExportTab.php (hunk 2 above)
Comment (jnmrclmbsjse) [reply src/StoreKeeper/WooCommerce/B2C/Backoffice/Pages/Tabs/ExportTab.php]: Used `esc_html__` for translations.

File: src/StoreKeeper/WooCommerce/B2C/Backoffice/Pages/Tabs/ExportTab.php (hunk 3 above)
Comment (jnmrclmbsjse) [reply src/StoreKeeper/WooCommerce/B2C/Backoffice/Pages/Tabs/ExportTab.php]: Used `esc_html__` for translations.

**Pull request merged**